py/torch_tensorrt/dynamo/Debugger.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import os
+import tempfile
+from logging.config import dictConfig
+from typing import Any, List, Optional
+
+import torch
+from torch_tensorrt.dynamo.lowering import (
+    ATEN_POST_LOWERING_PASSES,
+    ATEN_PRE_LOWERING_PASSES,
+)
+
+_LOGGER = logging.getLogger("torch_tensorrt [TensorRT Conversion Context]")
+GRAPH_LEVEL = 5
+logging.addLevelName(GRAPH_LEVEL, "GRAPHS")
+
+
+class Debugger:
+    def __init__(
+        self,
+        log_level: str,
+        capture_fx_graph_before: Optional[List[str]] = None,
+        capture_fx_graph_after: Optional[List[str]] = None,
+        save_engine_profile: bool = False,
+        logging_dir: Optional[str] = None,
+    ):
+        self.debug_file_dir = tempfile.TemporaryDirectory().name
+        if log_level != "graphs" and (capture_fx_graph_after or save_engine_profile):
+            _LOGGER.warning(
+                "Capture FX Graph or Draw Engine Graph is only supported when level is 'graphs'"
+            )
+
+        if log_level == "debug":
+            self.log_level = logging.DEBUG
+        elif log_level == "info":
+            self.log_level = logging.INFO
+        elif log_level == "warning":
+            self.log_level = logging.WARNING
+        elif log_level == "error":
+            self.log_level = logging.ERROR
+        elif log_level == "internal_errors":
+            self.log_level = logging.CRITICAL
+        elif log_level == "graphs":
```

Comment:
Cant remember if its GRAPH or GRAPHS in Torchscript but we should use the same name